manywheel/build_common.sh

```diff
@@ -100,7 +100,14 @@ if [[ ! -d "$pytorch_rootdir" ]]; then
 else
     pushd $pytorch_rootdir
 fi
-git submodule update --init --recursive --jobs 0
+
+if [[ -n "$GIT_JOBS_NUMBER" ]]; then
```

Comment:
1. can we rename to `GIT_UPDATE_NUM_JOBS` for clarity?
2. bash has a language feature for this pattern, could you switch to that to make this simpler? https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

Reply:
Sure, will rename it and use the recommend pattern, thanks for pointing out.